packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerHeader.tsx

```diff
@@ -4,30 +4,25 @@ import { from } from '@guardian/src-foundations/mq';
 import { headline } from '@guardian/src-foundations/typography';
 import { neutral } from '@guardian/src-foundations/palette';
 import { HeaderSettings } from '../settings';
-import useMediaQuery from '../../../../hooks/useMediaQuery';
 
 interface MomentTemplateBannerHeaderProps {
-    heading: JSX.Element | JSX.Element[] | null;
-    mobileHeading: JSX.Element | JSX.Element[] | null;
+    mainOrMobileContent: JSX.Element | JSX.Element[] | null;
     headerSettings: HeaderSettings | undefined;
 }
 
 export function MomentTemplateBannerHeader({
-    heading,
-    mobileHeading,
+    mainOrMobileContent,
     headerSettings,
 }: MomentTemplateBannerHeaderProps): JSX.Element {
-    const isTabletOrAbove = useMediaQuery(from.tablet);
-
     return (
         <div css={styles.container}>
             <header css={styles.header(headerSettings)}>
                 <h2>
-                    {headerSettings?.image
-                        ? headerSettings.image
-                        : isTabletOrAbove
-                        ? heading
-                        : mobileHeading}
+                    {// show headerText if a textColour supplied or headerImage missing
+                    headerSettings?.textColour || !headerSettings?.image
+                        ? mainOrMobileContent
+                        : null}
+                    {headerSettings?.image}
```

Comment:
I'm not sure it's a good idea to base whether we display the header copy on whether a certain css prop is present. It looks like the `textColour` prop will also now be optional and the `HeaderSettings` type it belongs to is also currently optional - but it might be a good idea to update this. 

I can see why this approach was taken i.e. to maintain the current visual display of existing moment template banner examples.
Displaying both a banner image and text would require an update to the `HeaderSettings` but I think it would be best to create an additional type, something to the effect of: 

```
showHeader {
  text: boolean;
  image?: boolean;
}
```

Nb: We'd also want to ensure changes to the `BannerTemplateSettings` are run by MT-TX, to ensure they wouldn't have an adverse affect on the banner tooling.

Reply:
Ok, have applied an explicit showHeader, will notify MT/TX now. Can reverse the commit if they don't want this change  inside `BannerTemplateSettings`